### ISSUE TITLE
Should voting records for members of the House of Lords be shown if they're not being kept up to date?

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/theyworkforyou#1407]
Recent votes in the House of Lords aren't being described on PublicWhip and systematically added to policies as is happening for House of Commons votes. 

TheyWorkForYou is generally handling this nicely; in that if a member of the House of Lords hasn't had the chance to take part in a vote on a subject nothing is said about their voting record on that subject.

Voting records aren't shown at all for those who've never had the chance to take part in a vote which is part of a tracked policy position. 

It could be misleading though to show a voting record which isn't being kept up to date; though the dates of the votes taken into account are shown prominently.

**Comment by MyfanwyNixon** [comment]
>It could be misleading though to show a voting record which isn't being kept up to date; though the dates of the votes taken into account are shown prominently.

Yes, I agree that not everyone will take notice of the dates even though they are there. 

I suppose the issue is that is a Lord's subsequent votes might outweigh the apparent stances displayed, as in the case of the peer who has contacted us to say votes display an apparent disregard for the topics in hand "despite the fact that he has been an advocate for these issues for many years, especially the last 5".

We should display an explanatory note at least, if not removing or updating them.
